installing/installing_aws/installing-aws-customizations.adoc

```diff
@@ -31,6 +31,10 @@ program.
 ====
 * If you use a firewall, you must
 xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configure it to allow the sites] that your cluster requires access to.
+* If you do not allow the system to manage identity and access management (IAM),
+then you can xref:../../installing/installing_aws/manually-creating-iam.adoc#manually-creating-iam[manually
+create IAM credentials]. In environments where the cloud IAM APIs are not
+reachable, the Cloud Credential Operator can be disabled prior to installation.
```

Comment:
instead of "cane be disabled", maybe "can be bypassed"?

Reply:
Updated with language with Devan's suggestion.